ci/check-documentation.py

```diff
@@ -114,8 +114,25 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
-        # check for existence of links to README.md files of models and demos
+        # check <omz_dir> link validity
+
+        for link in sorted([link for link in doc_page.omz_references() if link.startswith('<omz_dir>')]):
```

Comment:
The expected prefix should be `<omz_dir>/`, rather than `<omz_dir>`. If `<omz_dir>` is used by itself, then it's pointless to check it (it's obviously correct); and if it's followed by something other than a slash, then it's not a path and we shouldn't try to check it as a path.

Reply:
Also, I don't think the links should be sorted; it would be more useful for the developers to see the list of broken links in order of appearance.